src/Symfony/Component/Workflow/WorkflowInterface.php

```diff
@@ -35,12 +36,13 @@ public function getMarking($subject);
      *
      * @param object $subject        A subject
      * @param string $transitionName A transition
+     * @param bool   $throwException Throw an exception with some details if the transition is not enabled
      *
      * @return bool true if the transition is enabled
      *
-     * @throws LogicException
+     * @throws BlockedTransitionException
      */
-    public function can($subject, $transitionName);
+    public function can($subject, $transitionName, bool $throwException = false);
```

Comment:
Adding a new argument in the interface is a BC break, and cannot be done before 5.0

Reply:
The interface is not released yet. So there are no BC ;)

Reply:
then OK.
It thought it was already there in 4.0